Runtime/Scripts/ImportSettings.cs

```diff
@@ -48,5 +48,21 @@ public enum AnimationMethod
 
         public NameImportMethod nodeNameMethod = NameImportMethod.Original;
         public AnimationMethod animationMethod = AnimationMethod.Legacy;
+
+        /// <summary>
+        /// Set this property to true to enable mip map generation.
+        /// </summary>
+        public bool generateMipMaps = true;
+
+        /// <summary>
+        /// These two properties define the default filtering mode for textures that have no such specification in data
+        /// </summary>
+        public int defaultMinFilterMode = (int)Schema.Sampler.MinFilterMode.NearestMipmapLinear;

```

Comment:
I asked myself "Why not use the enum type (e.g.`Sampler.MinFilterMode`) instead of int?"
Then I figured you probably just followed along `Sampler.minFilter`, which is also int.
I don't like that, so I changed it in the main branch and wrote a test that certified JSON deserialization is still good.

Please re-base your work on branch main

Reply:
To clarify: Please use the more distinct enum types instead of generic int whenever possible.